pkg/optjson/optjson_test.go

```diff
@@ -88,3 +88,175 @@ func TestString(t *testing.T) {
 		}
 	})
 }
+
+func TestSlice(t *testing.T) {
```

Comment:
For the cases in this test, I think it would be helpful to have the struct field names in the literals to improve readability.

Reply:
good idea, added in https://github.com/fleetdm/fleet/pull/14190/commits/44129a5239e477df9406581de333411ebe5dac6a